storyscript/hub/sdk/ServiceWrapper.py

```
@@ -0,0 +1,132 @@
+                service_name = _service["service"]["name"]
+                self.services[service_owner + "/" + service_name] = _service
+
+    def as_json(self):
+        services = []
+
+        for service in self.services:
+            services.append(self.services[service])
+
+        return json.dumps(services, indent=4, sort_keys=True, cls=UUIDEncoder)
+
+    def as_json_file(self, out_file=None):
+        if out_file is not None:
+            with open(out_file, 'w') as f:
+                f.write(self.as_json())
+                f.close()
+
+    def get_all_service_names(self, include_aliases=True):
+        service_names = []
+
+        for service in self.services:
+            _service = self.services[service]["service"]
+            service_names.append(service)
+            alias = _service["alias"]
+
+            if include_aliases and alias is not None and alias not in service_names:
+                service_names.append(alias)
+
+        return service_names
+
+    def get(self, alias=None, owner=None, name=None):
+        service = None
+
+        if alias and alias in self.services:
+            service = self.services[alias]
+        elif f'{owner}/{name}' in self.services:
+            service = self.services[f'{owner}/{name}']
+        else:
+            for _service in self.services:
+                service_data = self.services[_service]["service"]
+                if owner is not None and name is not None and\
+                        service_data["owner"]["username"] == owner and \
+                        service_data["name"] == name:
+                    service = self.services[_service]
+                elif name is not None and service_data["name"] == name:
+                    service = self.services[_service]
+                elif alias is not None and (service_data["alias"] == alias or service_data["name"] == alias):
+                    service = self.services[_service]
+
+        if service is None:
+            return None
```

Should be an exception, not a silent failure.